server/cards/index.json

```diff
@@ -307,6 +306,16 @@
             "ribbon": {"text":"beta"},
             "sandboxJson": "databrowser",
             "searchterms": ["clinical", "dictionary"]
+        },
+        {
+            "type": "card",
+            "name": "Disco Plot",
+            "section": "apps",
+            "description": "Expression data arranged by genomic position",
+            "image": "https://proteinpaint.stjude.org/ppdemo/images/disco-square.png",
```

Comment:
this png file size is 150Kb, too big for an icon. please shrink size of all these png files, with tools like `optipng`